google-client-api.html

```diff
@@ -27,14 +27,24 @@
     </script>
 
 @element google-client-api
-@extends core-shared-lib
 @homepage https://googlewebcomponents.github.io/google-apis
 -->
-<polymer-element name="google-client-api" extends="core-shared-lib">
+<polymer-element name="google-client-api">
+  <template>
+    <google-js-api on-js-api-load="{{loadClient}}"></google-js-api>
+  </template>
 <script>
   Polymer({
-    defaultUrl: 'https://apis.google.com/js/client.js?onload=%%callback%%',
-    
+    /**
+     * Loads the client library when the api js has loaded.
+     */
+    loadClient: function() {
+      var self = this;
+      gapi.load("client", function() {
```

Comment:
Out of curiosity, this will incur 1 more network request for the client. Is that right? Is there still a way for users to load client.js directly